tasks/supply-chain-security/task.yaml

```diff
@@ -34,3 +34,19 @@ spec:
     echo $GIT_TOKEN | oras push -u $GIT_USER --password-stdin ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:$VERSION-sbom /workspace/source/sbom.json
     echo $GIT_TOKEN | oras push -u $GIT_USER --password-stdin ghcr.io/$DOCKER_REGISTRY_ORG/$REPO_NAME:latest-sbom /workspace/source/sbom.json
 
+  - name: install-grype
+    image: alpine:3.16
+    resources: {}
+    script: |
+      #!/bin/bash
+      apk add --no-cache curl unzip
+      curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | \
```

Comment:
Is there a way to verify that the binary is actually from grype? See codecov example: https://github.com/jenkins-x-plugins/jx-pipeline/blob/4950a36382b55f41a74f15b802e608340bfa4a9c/.lighthouse/jenkins-x/pullrequest.yaml#L35-L40

After all, this is all about supply chain!

Reply:
This is handled inside the installation script itself. 
Refer to the main function and it uses download_and_install_asset at https://github.com/anchore/grype/blob/98104952120d3e368d265566b3d6b415e9da538a/install.sh#L681
And if we continue following the structure of all functions used we reach the hash_sha256_verify defined at
https://github.com/anchore/grype/blob/98104952120d3e368d265566b3d6b415e9da538a/install.sh#L272
So, they are actually doing verification steps internally